docs/Releasing-the-plugin.md

```diff
@@ -0,0 +1,25 @@
+# Releasing the performance plugin
+
+This document describes the steps to release the Performance plugin.
+
+## Preparing the release
+
+> The following changes need to be made through a pull request.
+
+### Update the version number
+
+The version number needs to be updated in the following files:
+
+- package.json
+- load.php
+- readme.txt
+
```

Comment:
We are missing the step to update the module header translation strings (see #59), which could be added here:
```suggestion

### Update translation strings

The module headers from the plugin have to be translated using a separate `module-i18n.php` file which can be automatically generated and updated. Run `npm run translations` to update the file to reflect the latest available modules.

```